incidents/vub_20221118.yml

```diff
@@ -0,0 +1,12 @@
+meta_data:
+  title: VSC network unavailable
+  start_date: 2022-11-18 09:37:00
+  end_date:
+  affected: backbone, tier2_brussels
+  level: high
+  planned: no
```

Comment:
for some reason this goes up to the planned list when I look at https://status.vscentrum.be/

Reply:
I know. See #17

Reply:
if you look at https://status.vscentrum.be/backbone.html#incident_0003 it is planned, so there is a bug somewhere.

Reply:
ah, issue already logged at https://github.com/vscentrum/status-page/issues/17